<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTcpClientClass</class>
 <widget class="QMainWindow" name="MyTcpClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTcpClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>581</width>
      <height>109</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>109</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>109</height>
     </size>
    </property>
    <property name="title">
     <string>发送窗口</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="edtSend">
         <property name="minimumSize">
          <size>
           <width>460</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>31</y>
      <width>410</width>
      <height>322</height>
     </rect>
    </property>
    <property name="title">
     <string>接收窗口</string>
    </property>
    <widget class="QTextEdit" name="edtRecv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>390</width>
       <height>290</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>390</width>
       <height>290</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>150</width>
      <height>322</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1）服务器IP地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtIP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2）服务器端口号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>84</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>8000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>清除窗口</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyTcpClient.qrc"/>
 </resources>
 <connections/>
</ui>
